libs/net/dal/Migrations/Initial/Up/PostUp/02-Ingest.sql

```diff
@@ -463,7 +463,7 @@ INSERT INTO public.ingest (
   , 'GLOBE' -- topic
   , frontPageId -- product_id
   , '{ "path": "gandm/papers",
-      "fileName": "sv-GLB-<date>-.+?\.jpg",
+      "fileName": "sv-GLB-<date>-.+?.jpg",
```

Comment:
Why are you making these changes?  This is a regex expression and needs the leading escape character.

Reply:
Mac thing I guess... get an error about the `\` char when running script

Reply:
It can't be removed.  You also shouldn't be running this script.  The DB migration tool should be running this script.

Reply:
yeah i wasn't running it directly, it was failing regardless, can't recall on exactly which step, i'll revert the change 👍